admin-manual/customization/authentication.rst

```diff
@@ -0,0 +1,91 @@
+.. _customization-authentication:
+
+===================
+User authentication
+===================
+
+AtoM defaults to using local user authentication, which stores user details
+such as email addresses and passwords in the AtoM database. It is possible
+to configure AtoM to instead integrate with existing Single sign-on services
+using the `CAS <https://www.apereo.org/projects/cas>`_ or
+`LDAP <https://en.wikipedia.org/wiki/Lightweight_Directory_Access_Protocol>`_
+protocols. 
+
```

Comment:
Minor request - Can you add in a simple set of "Jump to" links for the major sub-sections here, and any relevant SEEALSO links? And then make the other parts subsections of a general CAS section (i.e. changing the titles from `==` underlines to `--` so they will display as subsections)? I just find this easier when sharing links in the user forum, for example, so people get a summary and related links at the top of sections :)

e.g. adding here:

```
**Jump to**:

* :ref`cas-enabling`
* :ref:`ldap-enabling`

.. SEEALSO::
   
   * :ref:`customization-config-files`
   * :ref:`manage-plugins`
   * :ref:`log-in`
   * :ref:`read-only-mode`
```

And then subheadings would be like, for example: 

```
.. _cas-groups:

Setting AtoM group membership from CAS user attributes
------------------------------------------------------
```